docs/source/conf.py

```diff
@@ -19,7 +19,6 @@
 author = "The National Library of Norway"
 email = "dh-lab@nb.no"
 copyright = f"{datetime.datetime.now().year}, {author}"
-# version = importlib.metadata.version("dhlab")
 version = "2.32.0"
```

Comment:
NB! Here, the commented code should actually be used, because the static version value is wrong (the latest version is 2.41.0). However, I had issues with importlib at the time, and used the hard-coded version to be able to build the docs. I'll add an issue to fix this.